### DISCUSSION TITLE
Keep axis functioning as analog?

### DISCUSSION THREAD

**Discussion by M1-Ace** [AntiMicroX/antimicrox#802] [Q&A]
I have all of my regular controller buttons (XYAB, D-Pad, shoulders) mapped to keyboard buttons, but I want to keep functionality with my triggers and joysticks. Right now they are mapped as [none] and give no output, and I cant seem to find a way to tell them to keep functioning normally.

**Comment by mrbeauregarde** [comment]
Your question is confusing to me.. are you saying that, for example, A is the result of pressing D_Right?
If I am understanding, you want,

> 1. Triggers to do what triggers do

... but there isn't an option for trigger press in the UI. The answer to the first problem is the answer to the question "what did triggers do?" I assume Left and Right Mouse click, if yes, instead of "none" bind that. And,

> 2. Joysticks to do what joysticks do

Second verse same as the first. The penultimate option under "L/R Stick" is mouse settings.

Hope I understood you right and this isn't just totally off-topic. gl
